lib/rouge/lexers/ada.rb

```diff
@@ -12,7 +12,7 @@ class Ada < RegexLexer
       desc 'The Ada 2012 programming language'
 
       # Ada identifiers are Unicode with underscores only allowed as separators.
-      ID = /\b[[:alpha:]](?:\p{Pc}?[[:alnum:]])*\b/
+      ID = /[[:alpha:]](?:\p{Pc}?[[:alnum:]])*\b/
```

Comment:
The word boundary start here is not a requirement of the lexer - so much so that there is a test that ensures it's not there. It was just... not working before.